src/main/java/com/minecolonies/util/ServerUtils.java

```diff
@@ -0,0 +1,50 @@
+package com.minecolonies.util;
+
+import java.util.List;
+import java.util.UUID;
+
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.entity.player.EntityPlayerMP;
+import net.minecraft.server.MinecraftServer;
+
+/**
+ * Utility for server related stuff
+ * 
+ * @author Isfirs
+ * @since 0.2
+ */
+public final class ServerUtils
+{
+
+    /**
+     * Private
+     */
+    private ServerUtils()
+    {
+    }
+
+    /**
+     * Found on <a href=
+     * "http://jabelarminecraft.blogspot.de/p/minecraft-forge-172-finding-block.html">jabelarminecraft</a>
+     * 
+     * @param parUUID
+     * @return The player
+     */
+    public static EntityPlayer getPlayerOnServerFromUUID(UUID parUUID)

```

Comment:
I suggest moving `EntityUtils#getPlayerFromUUID` in this class, `ServerUtils` sounds like the right place for this. Agreeing with colton on the reuse.